README.md

```diff
@@ -50,15 +50,22 @@ To clone the repository, complete the following steps:
 
 ## Get started
 
-For an overview of the user interface and features, refer to
-the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
-
-To review code samples and start building, refer to
-the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
-Codelab.
-
-To make sure that your device has the latest Matter support, review the
-[Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services) guide.
+*   For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+*   To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+*   To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   Build a Matter device with On/Off capabilities. This Codelab was tested with an ESP32 and a
+    virtual device.
+    *   [Building a Matter Virtual Device](https://developers.home.google.com/codelabs/matter-device-virtual)
+    *   [Build an Espressif Device](https://developers.home.google.com/matter/vendors/espressif)
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create).
+    For **Configuration**, use `0xFFF1` as your Vendor ID
+    (VID) and `0x8001` as your Product ID (PID). These are the ESP32 defaults
+    for the M5Stack `all-clusters-app`.
```

Comment:
We should provide more details about the matter-virtual-device we use for testing. 
Which one we build as well as its PID.

Reply:
Thanks - done.